src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -723,4 +726,35 @@ class EthService(
     }
 
   }
+
+  def getAccountRecentTransactions(request: GetAccountRecentTransactionsRequest): ServiceResponse[GetAccountRecentTransactionsResponse] = {
```

Comment:
I'm wondering, shouldn't we put this endpoint on a custom _"namespace"_ like `mantis_`? I'

Reply:
good question, I don't know. @rtkaczyk I think you mentioned Parity has some custom endpoints, are they in separate namespace?

Reply:
Yes, Parity has `parity_` 😄

Reply:
I'll put it in another namespace, but still leave it in EthService class (as it seems tightly coupled, it uses the same dto classes etc) wdyt?

Reply:
I think its ok. (When I need to look for an implementation I start from `JsonRpcContoller` so it's easy to figure out that will be on `EthService`)